Source/Core/PolygonGeometry.js

```diff
@@ -690,6 +690,10 @@ define([
             outerPositions[i] = ellipsoid.scaleToGeodeticSurface(outerPositions[i]);
         }
 
+        if (outerPositions.length < 3) {
```

Comment:
Actually, sorry, we can remove this check.  I didn't see there was a check above: [Line 684](https://github.com/TomPed/cesium/blob/fix-empty-rectangle-bug/Source/Core/PolygonGeometry.js#L684-L686)

The `polygonsFromHierarchy` function won't add polygons to the array if they have less than 3 positions.